am-lib/src/main/java/uk/gov/hmcts/reform/amlib/enums/AccessType.java

```diff
@@ -1,6 +1,5 @@
 package uk.gov.hmcts.reform.amlib.enums;
 
 public enum AccessType {
```

Comment:
we need to have a chat about the best solution for this. AccessManagamentType and AccessType is pretty unclear

Reply:
👍